Index: setup.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- setup.py	(date 1430494792000)
+++ setup.py	(revision )
@@ -1,6 +1,5 @@
-from distutils.core import setup
+from setuptools import setup
 import os, fnmatch
-import imp
 
 def find_package_data_files(directory):
     for root, dirs, files in os.walk(directory):
@@ -9,16 +8,9 @@
                 filename = os.path.join(root, basename)
                 if filename.find('/.svn') > -1:
                     continue
-                yield filename #.replace('cf/', '', 1)
+                # remove "cfunits" directory name for setuptools to appropriately copy the data files
+                yield filename[8:] #.replace('cf/', '', 1)
 
-# Check that the dependencies are met
-for _module in ('netCDF4', 'numpy'):
-    try:
-        imp.find_module(_module)
-    except ImportError as error:
-        raise ImportError, "Missing dependency. %s" % error
-#--- End: for
-
 version   = '0.9.6'
 etc_files = [f for f in find_package_data_files('cfunits/etc')]
 doc_files = [f for f in find_package_data_files('cfunits/docs')]
@@ -44,5 +36,5 @@
       package_data = {'cfunits': doc_files,
                       'cfunits': etc_files,
                       },
-      requires     = ['netCDF4 (>=0.9.3)', 'numpy (>=1.4)']
+      install_requires     = ['netCDF4>=0.9.3', 'numpy>=1.4']
-  )
+  )
\ No newline at end of file
Index: cfunits/cfunits.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- cfunits/cfunits.py	(date 1430494792000)
+++ cfunits/cfunits.py	(revision )
@@ -100,8 +100,8 @@
 # --------------------------------------------------------------------
 # Aliases for netCDF4.netcdftime functions
 # --------------------------------------------------------------------
-_num2date = netCDF4.netcdftime.num2date
-_date2num = netCDF4.netcdftime.date2num
+_num2date = netCDF4.num2date
+_date2num = netCDF4.date2num
 
 # --------------------------------------------------------------------
 # Save some useful units
